From a14123a609fc95f7e1f259fd53019468c5fb39fe Mon Sep 17 00:00:00 2001
From: Qi Deng <dengqi7@huawei.com>
Date: Thu, 11 Nov 2021 16:29:31 +0800
Subject: [PATCH] configs: add config BMA to config files
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: a14123a609fc95f7e1f259fd53019468c5fb39fe


driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4ETXO
CVE: NA

-----------------------------------------

Make CONFIG_BMA=m, except euleros_defconfig.

Link: https://lkml.org/lkml/2020/6/22/752

Signed-off-by: Qi Deng <dengqi7@huawei.com>
Reviewed-by: Qidong Wang <wangqindong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 arch/x86/configs/openeuler_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index cb9f3645d84c..7e6902f64c66 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -2649,6 +2649,7 @@ CONFIG_HNS3_HCLGEVF=m
 CONFIG_HNS3_ENET=m
 CONFIG_NET_VENDOR_HUAWEI=y
 CONFIG_HINIC=m
+CONFIG_BMA=m
 # CONFIG_NET_VENDOR_I825XX is not set
 CONFIG_NET_VENDOR_INTEL=y
 # CONFIG_E100 is not set
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 11e0d90b5f06..b25d908dc7a1 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -2676,6 +2676,7 @@ CONFIG_NET_VENDOR_GOOGLE=y
 # CONFIG_GVE is not set
 CONFIG_NET_VENDOR_HUAWEI=y
 CONFIG_HINIC=m
+CONFIG_BMA=m
 CONFIG_NET_VENDOR_NETSWIFT=y
 CONFIG_TXGBE=m
 # CONFIG_NET_VENDOR_I825XX is not set
-- 
2.26.2

